### PR TITLE
Removed build.sh scripts

### LINKED ISSUE #124: Remove old build scripts
The old `build.sh` scripts are deprecated now in favor of dbuild with `build.yml`.

Related to #84 since we really need docs for dbuild and our CI/CD pipeline in general.

### PR DESCRIPTION
**Pull request by kornicameister**
build.sh are no longer used. Local builds
can be executed with plain `docker build`
command therefore those scripts are not required.

Closes #124

### CODE REVIEW

**Comment by kornicameister** [comment]
@timothyb89 quess the reason it terminated....is....that the log output was too long :/. Anyway, I don't see this as the blocker, those scripts are, as far as I know, are pretty much redundant.

**Comment by timothyb89** [comment]
Yikes, I guess dbuild console output can get pretty spammy when lots of images are built at once. Those 
build scripts are definitely not used in CI. Rebuilding the containers isn't even necessary in this case, I believe we can just skip travis for this commit as a workaround with `[skip ci]`.

Also filed timothyb89/dbuild#3

File: monasca-python/build.sh [hunk 1]
```diff
@@ -1,122 +0,0 @@
-#!/bin/sh
```
Comment (timothyb89) [inline monasca-python/build.sh]: Oh wait a sec. Maybe this shouldn't be removed? :smiley:

Comment (kornicameister) [inline monasca-python/build.sh]: LOL, :1st_place_medal: for me.
I ran a ```find``` command and just removed everything that popped out...ah :D

**Comment by kornicameister** [comment]
Ok, I had to ``--force``` push, but I guess it is fine now.

**Pull request merged**